packages/amplify-graphql-schema-generator/src/ts-schema-generator/helpers.ts

```diff
@@ -46,17 +48,24 @@ const createDataType = (type: FieldType): ts.Node => {
     );
   }
 
+  // make it in the form a.ref('name') 
   if (type.kind === 'Enum') {
-    return ts.factory.createCallExpression(
-      ts.factory.createIdentifier(`${TYPESCRIPT_DATA_SCHEMA_CONSTANTS.REFERENCE_A}.${TYPESCRIPT_DATA_SCHEMA_CONSTANTS.ENUM_METHOD}`),
-      undefined,
-      [
-        ts.factory.createArrayLiteralExpression(
-          type.values.map((value) => ts.factory.createStringLiteral(value)),
-          true,
-        ),
-      ],
-    );
+    if(engine === 'Postgres'){
```

Comment:
Looking at the code without context, it looks like we don't handle enum types for MySQL engine. Since we clearly do, let's add a comment explaining the logic about why we're doing this.